portfolio/src/main/java/com/google/sps/servlets/ListDataServlet.java

```diff
@@ -40,11 +43,13 @@
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     int numCommentsToDisplay = getNumberOfComments(request);
 
-    Query query = new Query("Comment").addSort("timestamp", SortDirection.DESCENDING);
+    Filter niceComments = new FilterPredicate("sentimentScore", FilterOperator.GREATER_THAN_OR_EQUAL, new Double(-0.5));
```

Comment:
Instead of using -0.5 directly, I recommend having it stored in a private static variable of the class with a descriptive name.